rust-app/src/settings.rs

```diff
@@ -1,4 +1,6 @@
+#[cfg(target_os = "nanos")]
```

Comment:
Hmm maybe it would be good to split the module eventually? Or is the implementation less disjoint than it seems?